lib/jira/api.rb

```diff
@@ -62,7 +62,17 @@ def endpoint
     end
 
     def headers
-      { 'Content-Type' => 'application/json' }
+      args = {
+        'Content-Type' => 'application/json'
+      }
```

Comment:
I separated cookie to its own function in api.rb and it checks only for .empty? since cookie in core.rb will do the null check.